### PR TITLE
refactor: split Modify control in multiple interactions

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

**Review by friedjoff — APPROVED** [review]
Looks a lot cleaner and good to me!

**Review by danji90 — APPROVED** [review]
Looks great and works well. We could add some more Cypress tests in the future (e.g. for deleting features). But for now LGTM

**Review by danji90 — APPROVED** [review]
I think are done here now ;) LGTM
@oterral Don't forget to squash the commits on merge and please rename the PR to a conventional commit message (e.g. refactor(modify): separated Select, Delete...), because on squash it uses the PR title in the commit message

**Comment by oterral** [comment]
@danji90  What happens if i put a bad commit name after clicking squash/merge button, does it fail?

**Comment by danji90** [comment]
> @danji90 What happens if i put a bad commit name after clicking squash/merge button, does it fail?

Good point. I guess it will

File: index.html
Comment (mhaertwig) [inline index.html:29]: Why did you add the tabIndex? I would remove it to make the map navigable without previous selection.

**Review by mhaertwig — COMMENTED** [review]
Great job, thanks!!

File: index.html
Comment (oterral) [reply index.html:29]: without we can't have keyboard event (keyboard pan or delete)

Comment (oterral) [reply index.html]: I found a solution rtfm : https://github.com/geops/openlayers-editor/pull/201/commits/6e66d4896639f6f854e86ae6ce88d03f6ddf7598

**Pull request merged**
